OneCommunityGlobal/HGNRest
# Xiao F update teams page sorting part b and c

# Description
![CleanShot 2023-12-29 at 16 36 28](https://github.com/OneCommunityGlobal/HGNRest/assets/43768723/8baa048b-7b13-40b9-ab2c-a896b18d0489)

## Related PRS (if any):
This frontend PR is related to the [#1756](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1756) and [#1757](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1757) frontend PR.
To test this backend PR you need to checkout the above frontend PR.

## Main changes explained:
- Update file `src/controllers/teamController.js` to set new modifiedDatetime when a member is added or removed from a team

## How to test:
1. check into current branch
2. do `npm install`, `npm run build` and `npm start` to run this PR locally
3. Please see  [#1756](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1756) and [#1757](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1757) frontend PR's test instructions
